### ISSUE TITLE
Discovered telemetry sensors lost after power of 

### ISSUE THREAD

**Issue by krzysztofkuczek** [EdgeTX/edgetx#4374]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

After discovering CRSF sensor, everything seems to be ok, but after powering the radio off, and then powering it on again, the list of sensors is empty.

### Expected Behavior

The list of sensors should be the same after power cycle.

### Steps To Reproduce

1). Go to telemetry tab
2). Discover sensors
3). Stop discovering
4). Power cycle
5). Got o telemetry tab, the sensors list is empty,  but it should not be empty

The all available firmware version are affected including n build from 2.12.2023

### Version

Nightly (Please give date/commit below)

### Transmitter

RadioMaster Pocket

**Comment by PYJTERSTM32** [comment]
I have the same problem, after restarting I have to search for sensors again

**Comment by pfeerick** [comment]
Is the list empty, or is it simply that you need to un-collapse the list? If this arrow is pointing down, you need to press enter while it is highlighted (like it is in the screenshot) to un-collapse/expand the telemetry sensors list...

Collapsed:
![image](https://github.com/EdgeTX/edgetx/assets/5500713/09c699ac-3ea3-4359-93e8-2805a66d9205)

Expanded:
![image](https://github.com/EdgeTX/edgetx/assets/5500713/4fc82132-b0aa-40bf-b9f6-5254baccf02a)

**Comment by krzysztofkuczek** [comment]
> Is the list empty, or is it simply that you need to un-collapse the list? If this arrow is pointing down, you need to press enter while it is highlighted (like it is in the screenshot) to un-collapse/expand the telemetry sensors list...
> 
> Collapsed: ![image](https://private-user-images.githubusercontent.com/5500713/292071798-09c699ac-3ea3-4359-93e8-2805a66d9205.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMyNjE0ODYsIm5iZiI6MTcwMzI2MTE4NiwicGF0aCI6Ii81NTAwNzEzLzI5MjA3MTc5OC0wOWM2OTlhYy0zZWEzLTQzNTktOTNlOC0yODA1YTY2ZDkyMDUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMjJUMTYwNjI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmFlZjE5MjljZTY4YTNlMDJmNWFhODRmMzczYjI4OWY1NGI3MDE5MTYwMGRjNzYxMmQxNjgyOGJkZmY3OWZmYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.DokNG0_ogx4gvulkJ2jgFv0cVR2QGdHF0Nd9rt5P-js)
> 
> Expanded: ![image](https://private-user-images.githubusercontent.com/5500713/292071907-4fc82132-b0aa-40bf-b9f6-5254baccf02a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMyNjE0ODYsIm5iZiI6MTcwMzI2MTE4NiwicGF0aCI6Ii81NTAwNzEzLzI5MjA3MTkwNy00ZmM4MjEzMi1iMGFhLTQwYmYtYjlmNi01MjU0YmFjY2YwMmEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMjJUMTYwNjI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjQ3YjkyOGI2YTgzNzIwMDc1NjdlODVmNWJiOTQ5YzhmYTgyYzEzZTJjMWRiMmRiYWIyNDEwMWQxZTRhNzc4OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.EVatAEdJwLW2-FDFvezKp18rWTJ6t31-qH57Igm6k84)

Yes, you are right. The list is simple not expanded, and maybe it should be ;). Anyway it does not required to discover sensors every time, so it's quite ok.
Thank you
Kris
